src/api.js

```diff
@@ -400,6 +418,11 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         for (var field = 0; field < ref; field += 1) {
             switch (sqlite3_column_type(this.stmt, field)) {
                 case SQLITE_INTEGER:
+                    var getfunc = config && config.useBigInt
```

Comment:
Maybe instead of `config &&` add `config = config || {}` at the start of the function. This will make it easier to extend in the future.

Reply:
> Maybe instead of `config &&` add `config = config || {}` at the start of the function. This will make it easier to extend in the future.

I think I did miss something, I see an issue on minified files. When we pass config object ```{ useBigInt: true }``` , ```config.useBigInt ?  ....``` will always be evaluated as ```false``` since object property name ```useBigInt``` will be mangled during modification to something like ```l.Ty```. So, it will be something like ```l.Ty ? ....``` , of which ```Ty``` is not a property of ```l``` at run time. Any idea how to move forward on this?

Reply:
Yes, you should use `config["useBigInt"]`

Reply:
> Maybe instead of `config &&` add `config = config || {}` at the start of the function. This will make it easier to extend in the future.
Just pushed the changes

Reply:
> Maybe instead of `config &&` add `config = config || {}` at the start of the function. This will make it easier to extend in the future.

Just pushed the changes, thanks